### PR TITLE
Add Rule to assert/refute Include Of

### PR DESCRIPTION
**Pull request by duduribeiro**
Use `assert_instance_of` or `refute_instance_of` to check if a class is
 or is not an instance of some class.

assert:
 ```ruby
 assert_instance_of(String, 'rubocop-minitest')
 ```

 refute:
 ```ruby
 refute_instance_of(String, 'rubocop-minitest')
```

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -304,6 +304,30 @@ refute(object.respond_to?(some_method))
 refute_respond_to(object, some_method)
 ----
 
+=== Assert Instance Of [[assert-instance-of]]
+
+Use `assert_instance_of` to assert if the object is an instance of some class.
```
Comment (bbatsov) [inline README.adoc]: Prefer x over y

File: README.adoc [hunk 2]
```diff
@@ -304,6 +304,30 @@ refute(object.respond_to?(some_method))
...
+----
+# bad
+assert('rubocop-minitest'.instance_of?(String))
+# good
+assert_instance_of(String, 'rubocop-minitest')
+----
+
+=== Refute Instance Of [[refute-instance-of]]
+
+Use `refute_instance_of` to check if the object is not an instance of some class.
```
Comment (bbatsov) [inline README.adoc]: Same here

**Comment by duduribeiro** [comment]
@bbatsov can you re-review ? thanks 🍻

File: README.adoc [hunk 3]
```diff
@@ -304,6 +304,30 @@ refute(object.respond_to?(some_method))
 refute_respond_to(object, some_method)
 ----
 
+=== Assert Instance Of [[assert-instance-of]]
+
+Prefer `assert_instance_of(class, object)` over `assert(object.instance_of?(class))`
```
Comment (koic) [inline README.adoc]: Please add a period at the end of the sentence.

```diff
-Prefer `assert_instance_of(class, object)` over `assert(object.instance_of?(class))`
+Prefer `assert_instance_of(class, object)` over `assert(object.instance_of?(class))`.
```

File: README.adoc [hunk 4]
```diff
@@ -304,6 +304,30 @@ refute(object.respond_to?(some_method))
...
+----
+# bad
+assert('rubocop-minitest'.instance_of?(String))
+# good
+assert_instance_of(String, 'rubocop-minitest')
+----
+
+=== Refute Instance Of [[refute-instance-of]]
+
+Prefer `refute_instance_of(class, object)` over `refute(object.instance_of?(class))`
```
Comment (koic) [inline README.adoc]: Ditto.

**Comment by duduribeiro** [comment]
@koic I commited again.

**Pull request merged**